LandLord-core/src/main/java/biz/princeps/landlord/placeholderapi/LLExpansion.java

```diff
@@ -42,54 +37,71 @@ public String onPlaceholderRequest(Player player, String s) {
         if (player == null) {
             return null;
         }
-        int maxClaimPermission = pl.getPlayerManager().getMaxClaimPermission(player);
-        int landcount = wg.getRegionCount(player.getUniqueId());
-        IOwnedLand region = wg.getRegion(player.getLocation());
+        final int maxClaimPermission = pl.getPlayerManager().getMaxClaimPermission(player);
+        final int landCount = wg.getRegionCount(player.getUniqueId());
```

Comment:
Also unsure about the finals. I personally dont think that they really improve code readability at all and have any real benefit. But thats just my two cents.

Reply:
🤷 certainly a habit and I personally don't mind, if you insist I will remove these keywords.